src/Moryx.Communication.Mqtt.Endpoints/Moryx.Communication.Mqtt.Endpoints.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
```

Comment:
I like that this will actually be the first package from us with a readme on nuget 👍 Make sure to add the following here to have it packed

```xml
<PropertyGroup>
  <PackageReadmeFile>README.md</PackageReadmeFile>
</PropertyGroup>

<ItemGroup>
  <None Include="README.md" Pack="true" PackagePath="" />
</ItemGroup>
```